plugins/chainlink.Dockerfile

```diff
@@ -36,7 +36,7 @@ COPY --from=buildgo /go/bin/chainlink /usr/local/bin/
 COPY --from=buildgo /go/bin/chainlink-solana /usr/local/bin/
 ENV CL_SOLANA_CMD chainlink-solana
 COPY --from=buildgo /go/bin/chainlink-median /usr/local/bin/
-ENV CL_MEDIAN_CMD chainlink-median
+#ENV CL_MEDIAN_CMD chainlink-median #TODO BCF-2112 re-enable median plugin
```

Comment:
is BCF-2112 the right ticket number? i thought that was for recovery

Reply:
I left it as part of that open ticket, hoping to fix it in a quick follow-up. Think it should be a separate?

Reply:
no, it's fine, just checking